ui/dashboards/src/views/ViewDashboard/ViewDashboard.tsx

```diff
@@ -12,24 +12,23 @@
 // limitations under the License.
 
 import { useEffect } from 'react';
-import { BoxProps } from '@mui/material';
+import { Box, BoxProps } from '@mui/material';
 import { DashboardResource, getDefaultTimeRange } from '@perses-dev/core';
 import { useQueryString } from '@perses-dev/plugin-system';
-import { TimeRangeProvider, TemplateVariableProvider, DashboardProvider } from '../context';
+import { ErrorBoundary, ErrorAlert, combineSx } from '@perses-dev/components';
+import { TimeRangeProvider, TemplateVariableProvider, DashboardProvider } from '../../context';
 import { DashboardApp } from './DashboardApp';
 
-export interface ViewDashboardProps extends BoxProps {
+export interface ViewDashboardProps extends Omit<BoxProps, 'children'> {
```

Comment:
Just to make it impossible to wrap ViewDashboard around some other component?

Reply:
Yeah, good question. Before we allowed a `children` prop cause we were passing the footer into this component (which was kinda silly). But when the footer moved, this prop became unused and we weren't actually even rendering it any more once the `DashboardApp` component was added. So I just made sure to be explicit so folks don't think they can pass `children` to it and then get confused when the children don't render.

Reply:
Yay! That makes sense to me, I moved that footer I think